packages/server/lambdas/alfresco/search.ts

```diff
@@ -0,0 +1,72 @@
+import { ALBEvent, Context } from 'aws-lambda';
+import AWS from 'aws-sdk';
+import axios from 'axios';
+
+import { getRataExtraLambdaError } from '../../utils/errors';
+import { log } from '../../utils/logger';
+import { getUser, validateReadUser } from '../../utils/userService';
+
+let alfrescoAPIKey: string | null = null;
+let alfrescoAPIUrl: string | null = null;
+const alfrescoAPIKeyName = process.env.ALFRESCO_API_KEY_NAME || '';
+const alfrescoAPIUrlName = process.env.ALFRESCO_API_URL_NAME || '';
```

Comment:
URL you can get the value from CDK (which you actually already do). So need to call it name.